silq/instrument_interfaces/keysight/Keysight_81180A_interface.py

```diff
@@ -218,6 +219,8 @@ def setup(self, **kwargs):
         # targeted_pulse_sequence is the pulse sequence that is currently setup
         self.targeted_pulse_sequence = self.pulse_sequence
         self.targeted_input_pulse_sequence = self.input_pulse_sequence
+        assert self.pulse_sequence.final_delay >= 0.175e-6, \
+            f'Final pulse sequence delay {self.pulse_sequence.final_delay} must be >= 175ns.'
```

Comment:
I can't remember if final_delay can also be zero or not. Does anyone else know?

Reply:
I tested this as well and I did not find any restrictions in the pulse_sequence.final_delay elsewhere. I added this restriction because during my tests if the final delay was smaller than this time, the oscilloscope could not properly trigger on the sequence, which seemed like there are some timing/trigger issues there, for arbstudio this time seems to be 211ns, which I haven't added yet, but probably should as well. At the moment for my setup, I set the default value to 320ns instead of 2ms, which is used on every other setup in the group. This 2ms time is by the way the reason we did not notice this pulseblaster delay issue earlier - the pre-delay in the first pulse was always exactly 2ms and was not defined by the parameters. I think my 320ns is defined based on the cable length/instruments delay, so doubt that it is general for everyone, but by setting the limitation to 175ns I wanted at least to warn everyone that there should be some limitations or at least it should be additionally tested.

Reply:
Okay cool, yeah I think it makes sense that final delay must be larger than zero